doc/benchmarking.md

```diff
@@ -40,7 +40,7 @@ The output will look similar to:
 Help
 ---------------------
 
-    build/src/bench/bench_bitcoin -?
+    build/src/bench/bench_bitcoin -h
```

Comment:
Maybe this will be the last time we rewrite this.
Introduced as `-?`: b21244e0be5727f8c4e8c5de0a9aa2c597ae8ed2
Changed to `--help`: 05fdb97df46d0a0675b93e9791bd5d498e5e5117
Changed back to `-?`: d8513fe41102dcbfc05235f3b95e33eb1878f880
Now we are going back to `-h`: f0130ab1a1e65583637b6a362b879ea3253e7bb7.

Reply:
I'm fine with `--help` as well, but `-?` just isn't working on a mac